screens/profileScreen.js

```diff
@@ -1,16 +1,59 @@
 import React from 'react';
-import { StyleSheet, Text, View, Button } from 'react-native';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  ScrollView,
+  TouchableHighlight
+} from 'react-native';
+import mockdb from '../mockdb.json';
 
 export default class ProfileScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userData: null
+    };
+  }
```

Comment:
I think in CRNA there's [@babel/plugin-proposal-class-properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) installed. This means you can do something like:

```js
export default class ProfileScreen extends React.Component {
    state = {
      userData: null
    }
  }
}
```